scripts/gitlint/zephyr_commit_rules.py

```diff
@@ -95,7 +95,7 @@ class TitleStartsWithSubsystem(LineRule):
     def validate(self, title, _commit):
         regex = self.options['regex'].value
         pattern = re.compile(regex, re.UNICODE)
-        violation_message = "Commit title does not follow [subsystem]: [subject] (and should not start with literal subsys:)"
+        violation_message = "Commit title does not follow [subsystem]: [subject] (and should not start with literal subsys or treewide)"
```

Comment:
why? TitleStartsWithSubsystem is correct and exact.

Reply:
because now it's `TitleStartsWithSubsystemOrTreewide`

Reply:
ok but then why keep

```py
class TitleStartsWithSubsystem(LineRule):
    name = "title-starts-with-subsystem"
```

rathern than use a generic name?

Reply:
> ok but then why keep
> 
> ```python
> class TitleStartsWithSubsystem(LineRule):
>     name = "title-starts-with-subsystem"
> ```
Because this check does nothing but verify that your title starts with a subsystem?

Reply:
I guess it depends on how you read it - "starts with [a] sub-system" vs. "starts with the literal 'subsystem'".

Reply:
yeah I realized that that's probably  how it was meant to be read after Anas comment